app/config/sources.inc.php

```diff
@@ -659,7 +648,6 @@
         'firefox/includes/mwc_2015_schedule.lang' => $mwc_locales,
         'firefox/installer-help.lang'             => $firefox_locales,
         'firefox/ios.lang'                        => $ios_landing_page,
-        'firefox/new.lang'                        => $firefox_desktop_android,
```

Comment:
If you check the file, this array `$firefox_os_consumer` becomes unused, so we can get rid of this definition.